test/parallel/test-http-econnreset.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const net = require('net');
+
+var server = net.createServer((c) => {
+  c.end();
+});
+
+server.listen(() => {
+  const port = server.address().port;
+
+  http.get({port: port, path: '/', host: common.localhostIPv4})
```

Comment:
In this test you didn't put any whitespace inside the curly braces, but in the other test you did. Can we try to be consistent.